src/components/ChangelogDialog/ChangelogDialog.tsx

```diff
@@ -38,12 +40,19 @@ export function ChangelogDialog({
     disableBodyScrollLock = true,
     disableOutsideClick,
     onClose,
+    onOpen,
     onStoryClick,
+    onLinkClick,
 }: ChangelogDialogProps) {
     const idRef = React.useRef<number>();
     idRef.current = idRef.current || getNextId();
     const dialogCaptionId = `changelog-dialog-title-${idRef.current}`;
 
+    useEffect(() => {
+        if (!open) return;
```

Comment:
What the reason to add callback, if you pass `open` from outside anyway?

Reply:
Looks like there is no need to keep this code inside the component

Reply:
Yes, you are right.